api_tests/src/post.spec.ts

```diff
@@ -55,7 +55,18 @@ afterAll(() => {
   unfollows();
 });
 
-function assertPostFederation(postOne?: PostView, postTwo?: PostView) {
+async function assertPostFederation(postOne: PostView, postTwo: PostView) {
+  // Link metadata is generated in background task and may not be ready yet at this time,
+  // so wait for it explicitly. For removed posts we cant refetch anything.
+  postOne = await waitForPost(beta, postOne.post, res => {
+    return res == null || res?.post.embed_title != null;
+  });
```

Comment:
The alt_text issue might be a logical problem here. Its returning even if the res is null.

Reply:
The failing test is "No image proxying if setting is disabled" which doesnt use this function. The null check is necessary because theres a removed post which cant be fetched.

Reply:
hrm... gotta be somewhere else then. Otherwise all the other PRs would have alt_text issues too.

Reply:
It seems to happen specifically with this PR, so be careful with merging even if CI randomly passes.